lib/show_n_tell/talk.ex

```diff
@@ -21,7 +24,17 @@ defmodule ShowNTell.Talk do
   @doc false
   def changeset(talk, attrs) do
     talk
-    |> cast(attrs, [:title, :description, :estimated_duration, :date])
-    |> validate_required([:title, :description, :estimated_duration, :date])
+    |> cast(attrs, [:title, :description, :estimated_duration, :talk_date])
+    |> validate_required([:title, :description, :estimated_duration, :talk_date])
+  end
+
+  def list_talks() do
```

Comment:
Normal this would go in a [context](https://hexdocs.pm/phoenix/contexts.html), but I don't this is great for now at least.

Reply:
Yep, we're going to move to contexts later.